strongbox-data-service/src/main/java/com/orientechnologies/orient/object/jpa/OJPAObjectDatabaseTxPresistence.java

```diff
@@ -0,0 +1,93 @@
+package com.orientechnologies.orient.object.jpa;
+
+import static com.orientechnologies.orient.core.entity.OEntityManager.getEntityManagerByDatabaseURL;
+import static com.orientechnologies.orient.object.jpa.parsing.PersistenceXmlUtil.PERSISTENCE_XML;
+
+import java.net.URL;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.spi.PersistenceProvider;
+import javax.persistence.spi.PersistenceUnitInfo;
+import javax.persistence.spi.ProviderUtil;
+
+import com.orientechnologies.orient.core.entity.OEntityManager;
+import com.orientechnologies.orient.object.jpa.parsing.PersistenceXmlUtil;
+
+/**
+ * @author Sergey Bespalov
+ *
+ */
+public class OJPAObjectDatabaseTxPresistence implements PersistenceProvider
```

Comment:
I think the same applies to that class. Did you copied implementation from orientdb jar's and put your customization inside the code? If so it makes review process just impossible.